libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -1270,4 +1271,28 @@ int lua_DroneCAN_get_FlexDebug(lua_State *L)
 }
 #endif // HAL_ENABLE_DRONECAN_DRIVERS
 
+#if AP_GPS_ENABLED
+int lua_gps_inject_data(lua_State *L)
+{
+    binding_argcheck(L, 3);       // self + 2 args
+    luaL_checkudata(L, 1, "gps"); // require GPS object as first argument
+
+    const char *data = luaL_checkstring(L, 2);
```

Comment:
Use `luaL_checklstring` and drop the second argument? Pretty sure neither can return `nullptr`.